python_modules/libraries/dagster-dask/dagster_dask_tests/test_execute.py

```diff
@@ -35,209 +27,185 @@ def simple(_):
+            assert result.success
 
 
 @op(ins={"df": In(DataFrame)})
-def dask_solid(_, df):
+def dask_op(_, df):
     pass
 
 
-@pipeline(
-    mode_defs=[
-        ModeDefinition(
-            resource_defs={"io_manager": fs_io_manager},
-            executor_defs=[*default_executors, dask_executor],
-        )
-    ]
-)
-def dask_pipeline():
-    dask_solid()
+def dask_job() -> JobDefinition:
+    @job(
+        executor_def=dask_executor,
+    )
+    def job_def():
+        dask_op()
+
+    return job_def
 
 
 def test_dask():
     run_config = {
-        "solids": {
-            "dask_solid": {
+        "ops": {
+            "dask_op": {
                 "inputs": {
                     "df": {"read": {"csv": {"path": file_relative_path(__file__, "ex*.csv")}}}
                 }
             }
         }
     }
     with instance_for_test() as instance:
-        result = execute_pipeline(
-            ReconstructablePipeline.for_file(__file__, dask_pipeline.name),
+        with execute_job(
+            reconstructable(dask_job),
             run_config={
-                "execution": {"dask": {"config": {"cluster": {"local": {"timeout": 30}}}}},
+                "execution": {"config": {"cluster": {"local": {"timeout": 30}}}},
                 **run_config,
             },
             instance=instance,
-        )
-
-    assert result.success
-
-
-def test_execute_on_dask_local_without_io_manager():
-    with pytest.raises(DagsterUnmetExecutorRequirementsError):
-        with instance_for_test() as instance:
-            result = execute_pipeline(
-                reconstructable(dask_engine_pipeline),
-                run_config={
-                    "execution": {"dask": {"config": {"cluster": {"local": {"timeout": 30}}}}},
-                },
-                instance=instance,
-                mode="default",
-            )
-            assert result.result_for_node("simple").output_value() == 1
+        ) as result:
+            assert result.success
 
 
 @op(ins={"df": In(DataFrame)})
-def sleepy_dask_solid(_, df):
+def sleepy_dask_op(_, df):
     start_time = time.time()
     while True:
         time.sleep(0.1)
         if time.time() - start_time > 120:
             raise Exception("Timed out")
 
 
-@pipeline(
-    mode_defs=[
-        ModeDefinition(
-            resource_defs={"io_manager": fs_io_manager},
-            executor_defs=[*default_executors, dask_executor],
-        )
-    ]
-)
-def sleepy_dask_pipeline():
-    sleepy_dask_solid()
+def sleepy_dask_job() -> JobDefinition:
+    @job(
+        executor_def=dask_executor,
+    )
+    def job_def():
+        sleepy_dask_op()
+
+    return job_def
 
 
+@pytest.mark.skip(
+    """
+    Fails because 'DagsterExecutionInterruptedError' is not actually raised-- there's a timeout
+    instead. It's not clear that the test ever was working-- prior to conversion to op/job/graph
+    APIs, it appears to have been mistakenly not using the dask executor. 
+    """
```

Comment:
good flag - not a blocker for this pr but should raise in eng-help and see if we can just delete.